WcaOnRails/config/initializers/translations.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+if defined?(Rails::Server) # See: https://stackoverflow.com/a/44441168
+  Rails.configuration.after_initialize do
+    modification_timestamp = Timestamp.find_or_create_by!(name: 'en_translation_modification')
+    latest_modification_date = DateTime.parse(`git log -1 --format='%ai' #{Rails.root.to_s}/config/locales/en.yml`)
+    if modification_timestamp.date != latest_modification_date
```

Comment:
WIth a reservation for me not knowing much about this area, that seems OK.

1. If you roll back a change in English that someone has already translated, the translators _should_ be notified.

2. This seems quite rare. The occasional false alarm is probably not a big deal?